print_nanny_webapp/users/models.py

```diff
@@ -181,3 +185,31 @@ class SiteChoices(models.TextChoices):
     email_opened_count = models.IntegerField()
     email_open_rate = models.FloatField(null=True)
     subscribed = models.BooleanField(default=True)
+
+
+class GeeksToken(models.Model):
+    # Grabbed code from rest_framework Token model
+
+    key = models.UUIDField(primary_key=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
ooooh, makes sense. so I'll just replace `settings.AUTH_USER_MODEL`, not the whole `user`, right?
Also as a side note, drf uses `settings.AUTH_USER_MODEL`, no idea if it's just "leftover" or actually makes any difference

will test and report back

Reply:
I guess that's not it. When I tried `get_user_model()` instead of `AUTH_USER_MODEL`  I got:
(maybe I can use the `User` class defined right above the Geeks class?)
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/apps/config.py", line 178, in get_model
    return self.models[model_name.lower()]
KeyError: 'user'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/contrib/auth/__init__.py", line 157, in get_user_model
    return django_apps.get_model(settings.AUTH_USER_MODEL, require_ready=False)
  File "/usr/local/lib/python3.8/site-packages/django/apps/registry.py", line 211, in get_model
    return app_config.get_model(model_name, require_ready=require_ready)
  File "/usr/local/lib/python3.8/site-packages/django/apps/config.py", line 180, in get_model
    raise LookupError(
LookupError: App 'users' doesn't have a 'User' model.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "manage.py", line 31, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 377, in execute
    django.setup()
  File "/usr/local/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.8/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/usr/local/lib/python3.8/site-packages/django/apps/config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/app/print_nanny_webapp/users/models.py", line 23, in <module>
    UserModel = get_user_model()
  File "/usr/local/lib/python3.8/site-packages/django/contrib/auth/__init__.py", line 161, in get_user_model
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: AUTH_USER_MODEL refers to model 'users.User' that has not been installed
```

Reply:
yeap, it was `User`